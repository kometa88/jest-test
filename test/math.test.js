const { sum, multiply } = require('../src/math');

describe('Math functions', () => {
  test('sum adds two numbers correctly', () => {
    expect(sum(1, 2)).toBe(3);
    expect(sum(-1, 1)).toBe(0);
    expect(sum(0, 0)).toBe(0);
  });

  test('multiply multiplies two numbers correctly', () => {
    expect(multiply(2, 3)).toBe(6);
    expect(multiply(-2, 3)).toBe(-6);
    expect(multiply(0, 5)).toBe(0);
  });

  test('sum function snapshot', () => {
    expect(sum(5, 5)).toMatchSnapshot();
  });
});