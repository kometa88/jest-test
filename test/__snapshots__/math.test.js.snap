// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Math functions sum function snapshot 1`] = `10`;
